subsys/nfc/ndef/Kconfig

```diff
@@ -10,6 +10,12 @@ config NFC_NDEF
 	help
 	  Enable NFC Data Exchange Format libraries
 
+config NFC_NDEF_PARSER
+	bool
+	prompt "NDEF parser libraries"
```

Comment:
`prompt` is redundant. Place the text next to `bool`. For consistency with other Kconfig files.

Reply:
Ok